css/colourpicker.css

```diff
@@ -36,36 +42,59 @@
   left: 0;
   top: auto;
 }
+
+
 .palette-square .colourpicker-panel {
-  width: 173px;
+  width: 174px;
   height: 152px;
 }
+
+.palette-square.colourpicker-with-alpha .colourpicker-panel {
+  width: 196px;
+}
+
 .palette-limited .colourpicker-panel {
-  width: 181px;
+  width: 196px;
   height: auto;
-  padding: 5px;
+  padding-left: 0px;
+  padding-right: 0px;
+  padding-top: 1px;
+  padding-bottom: 4px;
```

Comment:
If all 4 are defined, could you merge them into one statement `padding: 1px 0 4px 0;`